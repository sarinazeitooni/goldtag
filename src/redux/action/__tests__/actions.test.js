import React from 'react';
import { shallow } from 'enzyme';
import exportObject from "../actions";
describe("exportObject text" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<exportObject/>)
        expect(container).toMatchSnapshot();
    });
});