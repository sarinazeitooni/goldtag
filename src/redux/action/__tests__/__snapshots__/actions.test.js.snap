// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exportObject text it should render a div 1`] = `ShallowWrapper {}`;
